packages/flutter_tools/templates/app/ios.tmpl/.gitignore

```diff
@@ -18,6 +18,7 @@ Flutter/App.framework
 Flutter/Flutter.framework
 Flutter/Flutter.podspec
 Flutter/Generated.xcconfig
+Flutter/ephemeral/
```

Comment:
\cc @RedBrogdon Sorry here's another directory to add to all the iOS sample `.gitignore`s.  But new ones should be rare in the future with this PR.